src/Parser/Paladin/Protection/Modules/Features/AlwaysBeCasting.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import CoreAlwaysBeCasting from 'Parser/Core/Modules/AlwaysBeCasting';
+
+import SPELLS from 'common/SPELLS';
+import { formatPercentage } from 'common/format';
+import { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class AlwaysBeCasting extends CoreAlwaysBeCasting {
+  static ABILITIES_ON_GCD = [
+    SPELLS.ARDENT_DEFENDER.id,
```

Comment:
I'm not very involved in WoWA lately so I probably miss out on some crucial info, but wasn't there some issue with spells that are off CD in ABC? That it screws up with the calculations?

Reply:
Yep gotta remove this one